tests/globalhelper/statefulset.go

```diff
@@ -0,0 +1,40 @@
+package globalhelper
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/golang/glog"
+	. "github.com/onsi/gomega"
+	v1 "k8s.io/api/apps/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// CreateAndWaitUntilStatefulSetIsReady creates statefulset and waits until all its replicas are up and running.
+func CreateAndWaitUntilStatefulSetIsReady(statefulSet *v1.StatefulSet, timeout time.Duration) error {
```

Comment:
Same as above.  Move function from lifehelper.go to global level

Reply:
Same here. Can I do this refactor in a new PR, please?